### PR TITLE
Opt @foolip and @autofoolip out of staging.wpt.fyi checks

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
Sent this because I don't like seeing so many checks, and since things stabilized I never used the staging checks.

File: api/checks/webhook.go [hunk 1]
```diff
@@ -338,9 +338,7 @@ func isUserWhitelisted(aeAPI shared.AppEngineAPI, login string) bool {
 		return true
 	}
 	whitelist := []string{
-		"autofoolip",
 		"chromium-wpt-export-bot",
-		"foolip",
 		"gsnedders",
```
Comment (foolip) [inline api/checks/webhook.go]: @gsnedders @jgraham @jugglinmike @lukebjerring @Ms2ger FYI I'm opting myself out of staging checks. If anyone else doesn't want to see the extra checks either I can roll that into this PR.

**Comment by lukebjerring** [comment]
FWIW, staging actually performs a different computation of regressions (using the searchcache).

**Comment by foolip** [comment]
Oh, I hadn't noticed. Does it catch more cases?

**Comment by lukebjerring** [comment]
Less, actually :) it requires PASS => !PASS

**Comment by foolip** [comment]
Nah, I'll opt out and see if I regret it :)

**Pull request merged**